game_api.txt

```diff
@@ -227,6 +227,12 @@ New node def property:
  * Called when fire attempts to remove a burning node.
  * `pos` Position of the burning node.
 
+ `on_ignite(pos, igniter)`
+
+  * Called when a Flint and steel tool is used in a node.
```

Comment:
'On' instead of 'in':
`* Called when a Flint and steel tool is used on a node.`